src/css/main.css

```diff
@@ -1,5 +1,5 @@
-html, body, #cesiumContainer {
-	width: 100%;
+html, body, #cesiumContainer {	
+    width: 100%;
```

Comment:
This file has some inconsistent formatting with spaces and tabs. Could you clean it up?